cost/aws/old_snapshots/aws_delete_old_snapshots.pt

```diff
@@ -443,31 +437,34 @@ script "js_snapshots_cost_mapping", type: "javascript" do
       var monthly_savings = sum*30;
       total=total+monthly_savings;
       result.push({
-        accountId: ds_get_caller_identity[0]['account'],
+        accountID: ds_get_caller_identity[0]['account'],
         snapshotId : snapshot['snapshotId'],
         region : snapshot['region'],
-        tagKeyValue : snapshot['tagKeyValue'],
+        tags : snapshot['tagKeyValue'],
         daysOld : snapshot['daysOld'],
         imageId : snapshot['imageId'],
         snapshotImageIdFormat : snapshot['snapshotImageIdFormat'],
-        savings : cur + ' '+formatNumber(monthly_savings.toFixed(3), separator),
-        volumeSize : snapshot['volumeSize']
+        savings : parseFloat(monthly_savings.toFixed(3)),
```

Comment:
Rather than rendering to a string and then parsing it back to a number, it would probably be better to do something like:

```javascript
Math.round(monthly_savings * 1000) / 1000;
```

Reply:
good catch with it no longer being string. added these and standardized it at 3 decimals for all of the templates